src/components/Page/Page.tsx

```diff
@@ -94,7 +95,20 @@ export class Page extends React.PureComponent<ComposedProps, never> {
     );
   }
 
-  private hasHeaderContent() {
+  private get renderWithAppBridge(): boolean {
```

Comment:
actually just update this whole method, also I prefer `delegateToAppBridge` because we have no concerns with how app bridge handles the action call.

```suggestion
  private get delegateToAppBridge(): boolean {
    const {
      polaris: {appBridge},
      forceRender = false,
    } = this.props;
    
    return appBridge != null && forceRender === false;
  }
```